README.md

```diff
@@ -80,6 +80,19 @@ A high level breakdown of the components of the framework:
 * [mgfxc](Tools/MonoGame.Effect.Compiler) is the command line effect compiler tool.
 * The [mgcb-editor](Tools/MonoGame.Content.Builder.Editor) tool is a GUI frontend for content processing.
 
+### Syncing/Submitting PRs
+
+After the initial setup, use the following commands to sync to the latest changes.
+
+```bash
+git pull origin develop
```

Comment:
You actually wanna sync by doing `git pull --rebase origin develop` (I usually just call `git rebase -u origin develop` instead, might have made a mistake with the rebase version of pull command). This way your commits go on top of the upstream commits, and you don't create the "Merge branch develop of upstream" commits added.

This does however mean that pushing your changes back to your fork will require you to force push them.

Reply:
Just seeing this now. Thanks for the `rebase upstream / origin` tip :)